api/src/pcapi/core/subscription/ubble/api.py

```diff
@@ -453,15 +452,26 @@ def recover_pending_ubble_applications(dry_run: bool = True) -> None:
     from pcapi.core.subscription.ubble import tasks as ubble_tasks
 
     if FeatureToggle.WIP_ASYNCHRONOUS_CELERY_UBBLE.is_active():
-        stale_ubble_fraud_check_ids = _get_stale_fraud_checks_ids(CELERY_PAGE_SIZE)
-        for fraud_check_id in stale_ubble_fraud_check_ids:
+        started_fraud_checks = 0
+        pending_fraud_checks = 0
+
+        page_size = ubble_tasks.UBBLE_TASK_RATE_LIMIT * 5
+        stale_ubble_fraud_check_ids = _get_stale_fraud_checks_id_and_status(page_size)
+        for fraud_check_id, fraud_check_status in stale_ubble_fraud_check_ids:
             ubble_tasks.update_ubble_workflow_task.delay(
                 payload=ubble_schemas.UpdateWorkflowPayload(beneficiary_fraud_check_id=fraud_check_id).model_dump()
             )
 
+            if fraud_check_status == subscription_models.FraudCheckStatus.STARTED:
+                started_fraud_checks += 1
+            if fraud_check_status == subscription_models.FraudCheckStatus.PENDING:
+                pending_fraud_checks += 1
+
         logger.warning(
-            "Found %d stale ubble application older than 12 hours and tried to update them.",
+            "Found %d stale ubble application with %d currently started and %d currently pending",
             len(stale_ubble_fraud_check_ids),
+            started_fraud_checks,
+            pending_fraud_checks,
         )
```

Comment:
Est-ce qu'on ne loggerait pas ainsi ? L'idée c'est de pouvoir facilement faire une recherche dans l'explorer juste en cherchant `"Found stale ubble applications"`. Je me demandais aussi si ça serait pertinent d'avoir les ids dans l'objet `extra` (ça ne l'est peut-être pas du tout) 

```suggestion
        logger.warning(
            "Found stale ubble applications: %d currently started and %d currently pending",
            started_fraud_checks,
            pending_fraud_checks,
            extra={"stale_ubble_fraud_check_ids": stale_ubble_fraud_check_ids}
        )
```